skimage/data/tests/test_data.py

```diff
@@ -167,10 +165,6 @@ def test_brain_3d():
     assert image.shape == (10, 256, 256)
 
 
-@pytest.mark.xfail(
-    Version(np.__version__) >= Version('2.0.0.dev0'),
-    reason='tifffile uses deprecated attribute `ndarray.newbyteorder`',
-)
```

Comment:
I guess this issue https://github.com/cgohlke/tifffile/issues/238 was addressed in [tifffile 2024.1.30](https://github.com/cgohlke/tifffile/blob/master/CHANGES.rst). 

Should we update to tifffile 2024.1.30? I guess it might be possible that there are some combinations of our dependencies that still raise this warning or even fail but probably rare and fix-able with bumping tifffile or downgrading NumPy. 

So I think we can leave tifffile be.